### PR TITLE
Update liquid_use_cases.md with 30 for 30 tips

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 1]
```diff
@@ -110,6 +111,23 @@ Message body
...
 **Explanation:** Similar to the [birthday week](#birthday-week) use case, except here we use the `%B` filter (month, i.e., "May") to calculate which users have a birthday this month. A potential application could be addressing birthday users in a monthly email.
 
+### Avoid sending messages on major holidays {#holiday-avoid}
+
+This use case shows how to send messages during the holiday period while avoiding the days of major holidays, when engagement is likely to be low.
+
+{% raw %}
+```liquid
+{% assign today = 'now' | date: '%Y-%m-%d' %}
+{%if today == "2021-12-24" or today == "2021-12-25" or today == "2021-12-26” %}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:121]: Can you add a space between the percentage sign and the "if", "abort_message", and "endif"

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 2]
```diff
@@ -110,6 +111,23 @@ Message body
 
 **Explanation:** Similar to the [birthday week](#birthday-week) use case, except here we use the `%B` filter (month, i.e., "May") to calculate which users have a birthday this month. A potential application could be addressing birthday users in a monthly email.
 
+### Avoid sending messages on major holidays {#holiday-avoid}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:114]: Capitalize the use case title.

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 3]
```diff
@@ -760,6 +781,33 @@ Miscellaneous
 - [Query the end of a string](#misc-query-end-of-string)
 - [Query values in an array from a custom attribute with multiple combinations](#misc-query-array-values)
 
+### Avoid sending emails to customers that have blocked marketing emails {#misc-avoid-blocked-emails}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:784]: Capitalize use case title.

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Hey! Looks great, just made a few minor suggestions. Thanks!

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:121]: Good catch, thanks!!

**Review by KellieHawks — APPROVED** [review]
Looks great. Thanks for adding this in.

**Pull request merged**